src/pages/ReimbursementAccount/Enable2FAPrompt.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {View} from 'react-native';
+import Text from '../../components/Text';
+import styles from '../../styles/styles';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import * as Expensicons from '../../components/Icon/Expensicons';
+import * as Illustrations from '../../components/Icon/Illustrations';
+import Section from '../../components/Section';
+import * as Link from '../../libs/actions/Link';
+import CONFIG from '../../CONFIG';
+import ROUTES from '../../ROUTES';
+import themeColors from '../../styles/themes/default';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+};
+const Enable2FAPrompt = props => (
+    <Section
+        title={props.translate('validationStep.enable2FATitle')}
+        icon={Illustrations.ShieldYellow}
+        menuItems={[
+            {
+                title: props.translate('validationStep.secureYourAccount'),
+                onPress: () => {
+                    Link.openOldDotLink(`settings?param={"section":"account","action":"enableTwoFactorAuth","exitTo":"${CONFIG.EXPENSIFY.NEW_EXPENSIFY_URL}${ROUTES.BANK_ACCOUNT}"}`);
+                },
+                icon: Expensicons.Shield,
+                shouldShowRightIcon: true,
+                iconRight: Expensicons.NewWindow,
+                iconFill: themeColors.success,
```

Comment:
👋 Heads-up that this has caused a small visual issue in https://github.com/Expensify/App/issues/20932
This icon wasn't changing color on hover, since we were overriding its color with `iconFill` prop